yourls/docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+services:
+  yourls:
+    build:
+      context: yourls
+      args:
+        YOURLS_IMAGE: ${YOURLS_IMAGE}
+    restart: unless-stopped
+    depends_on:
+      - mysql
+    environment:
+      YOURLS_DB_PASS: ${MYSQL_ROOT_PASSWORD}
+      YOURLS_SITE: https://${YOURLS_TRAEFIK_HOST}
+      YOURLS_USER: ${YOURLS_USER}
+      YOURLS_PASS: ${YOURLS_PASS}
+      YOURLS_UNIQUE_URLS: ${YOURLS_UNIQUE_URLS}
+      YOURLS_COOKIEKEY: ${YOURLS_COOKIEKEY}
+      YOURLS_USER_PASSWORDS: ${YOURLS_USER_PASSWORDS}
```

Comment:
I dont see this var defined anywhere?

Reply:
It's explained [here](https://yourls.org/docs/guide/essentials/configuration#yourls_cookiekey). 

`make config` assigns a random value:
```
	@${BIN}/reconfigure_password ${ENV_FILE} YOURLS_COOKIEKEY 45
```

Reply:
fixed in [2dd4f1a](https://github.com/EnigmaCurry/d.rymcg.tech/pull/280/commits/2dd4f1a8ff212abf77e3394c1b825498f9b8b47a)